userguide/tasks.rst

```diff
@@ -285,8 +285,8 @@ task.
    +------------------------------+----------------+-------------------------------------------------------------------------------------------+
    | Setting                      | Value          | Description                                                                               |
    +==============================+================+===========================================================================================+
-   | Path                         | browse button  | :guilabel:`Browse` to the path to be copied. The FreeBSD                                  |
-   |                              |                | :ref:`file path limits <Path and Name Lengths>` apply.                                    |
+   | Path                         | browse button  | :guilabel:`Browse` to the path to be copied. %brand% verifies if the                      |
+   |                              |                | remote path exists.                                                                       |
```

Comment:
This should still mention: FreeBSD <path length limits> apply on the FreeNAS/TrueNAS system. Other operating systems can have different limits which might affect how they can be used as sources or destinations.